README.md

```diff
@@ -46,6 +46,7 @@ Conference | Dates | Proposals | Location | Twitter
 [DevFest Siberia](https://gdg-siberia.com) | Nov 23rd - 25th | [Jul 31st](https://cfp.exchange/cfp/details/devfest-siberia-2018) | Novosibirsk, Russia
 [NodeFest](https://nodefest.jp/2018/) | Nov 23th - 24th | [Sep 30th](https://docs.google.com/forms/d/1b3DFCCVIK1rinVMFO39q6hSUev1Q_27avmYRE2E4YG0/edit) | Tokyo, Japan | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/nodefest)
 [JSConf Colombia](https://jsconf.co) | Nov 16th - 17th | [Jul 31st](https://sessionize.com/jsconfco) | Medellin, Colombia | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/jsconfco)
+[JSDC-TW](https://js-poland.pl/) | Nov 17th | [Unknown](#) | Taipei, Taiwan | [![](https://user-images.githubusercontent.com/10602/44954380-6a3ab480-aea1-11e8-8ce7-ad540063f4de.png)](https://twitter.com/jsdc_tw)
```

Comment:
I think the url should be http://2018.jsdc.tw/ ?

Reply:
Good point 😃 

Two other tiny nits: Can you move the line up above JSConf Colombia and just remove the link around "Unknown"? Then we're good 👌 

Thanks for contributing 💯

Reply:
already updated, please check it, thanks